include/exadg/time_integration/time_int_gen_alpha_base.h

```diff
@@ -53,17 +53,26 @@ class TimeIntGenAlphaBase : public TimeIntBase
   set_current_time_step_size(double const & time_step_size) final;
 
   double
-  get_scaling_factor_mass() const;
+  get_scaling_factor_mass_from_acceleration() const;
+
+  double
+  get_scaling_factor_mass_from_velocity() const;
 
 protected:
   double
   get_mid_time() const;
 
   void
-  compute_const_vector(VectorType &       const_vector,
-                       VectorType const & displacement_n,
-                       VectorType const & velocity_n,
-                       VectorType const & acceleration_n) const;
+  compute_const_vector_acceleration_remainder(VectorType &       const_vector,
+                                              VectorType const & displacement_n,
+                                              VectorType const & velocity_n,
+                                              VectorType const & acceleration_n) const;
```

Comment:
I think we should add comments here (and below) that these "const vectors" do not appear directly as vectors in the residual (nonlinear) or rhs (linear) of the system of equations, but that the mass operator (or damping operator below) still need to be applied to these vectors.

Reply:
Good suggesstion, makes their intended use a lot clearer! I will add some comments.